cores/cv32e40p/user_manual/source/instruction_set_extensions.rst

```diff
@@ -1,23 +1,23 @@
 .. _custom-isa-extensions:
 
-PULP Instruction Set Extensions
-===============================
+CORE-V Instruction Set Extensions
+=================================
 
-CV32E40P supports the following PULP ISA Extensions, which are part of **Xpulp** and can be enabled by setting ``PULP_XPULP`` == 1.
+CV32E40P supports the following CORE-V ISA Extensions, which are part of **Xcorev** and can be enabled by setting ``PULP_XPULP`` == 1. 
```

Comment:
The RISC-V standard defines the ISA naming convention in Chapter 27, with non-standard extensions defined in section 27.10.  That document uses the typography `Xname`, but also specifies (section 27.1) that the names are case insensitive, so in GCC or Clang/LLVM `-march=rv32imcXcorev` and `-march=RV32IMCXCOREV` are the same thing.

I think there is no problem with keeping the internal parameters using whatever you wish.  The standard only defines what is seen externally.

Reply:
Hi @jeremybennett , yes I agree that there is no problem - it would have been nicer but now it's too late to change the parameter names for such a "non important" issue.